packages/web-components/src/anchored-region/index.ts

```diff
@@ -0,0 +1,25 @@
+import { customElement } from '@microsoft/fast-element';
+import { AnchoredRegion, AnchoredRegionTemplate as template } from '@microsoft/fast-foundation';
+import { AnchoredRegionStyles as styles } from './anchored-region.styles';
+
+/**
+ * The FAST AnchoredRegion Element. Implements {@link @microsoft/fast-foundation#AnchoredRegion},
+ * {@link @microsoft/fast-foundation#AnchoredRegionTemplate}
+ *
+ *
+ * @beta
```

Comment:
This control doesn't appear to be exported. Is it internal or is it beta?

Reply:
Updated, now exported. 
According to Steph it is still beta. folks can use it, but shouldn't be surprised by changes.